.github/workflows/build-testbenches.yml

```diff
@@ -0,0 +1,61 @@
+name: Build testbenches
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+env:
+  CROSS_CC_PREFIX: riscv-none-elf-
+  CROSS_CC_URL: https://github.com/xpack-dev-tools/riscv-none-elf-gcc-xpack/releases/download/v14.2.0-1/xpack-riscv-none-elf-gcc-14.2.0-1-linux-x64.tar.gz
```

Comment:
Let's maintain the rules to fetch and extract prebuilt GNU Toolchain via the scripts in `.ci` directory.